packages/@uppy/companion/src/standalone/helper.js

```diff
@@ -57,6 +57,11 @@ const getConfigFromEnv = () => {
         region: process.env.COMPANION_AWS_REGION,
         useAccelerateEndpoint:
           process.env.COMPANION_AWS_USE_ACCELERATE_ENDPOINT === 'true'
+      },
+      b2: {
+        keyId: process.env.COMPANION_B2_KEY_ID,
```

Comment:
This is used as `applicationKeyId` elsewhere:
```suggestion
        applicationKeyId: process.env.COMPANION_B2_KEY_ID,
```

Reply:
Hah yeah, renaming all the other things works too :wink: